pkgs/development/python-modules/proxmoxer/default.nix

```diff
@@ -36,6 +37,7 @@ buildPythonPackage rec {
     pytestCheckHook
     requests-toolbelt
     responses
+    pynacl
```

Comment:
This was already with https://github.com/NixOS/nixpkgs/commit/50d865aa192645e10064173422b394a12aecdfea and is in `staging-next`.

Reply:
Do you think this doesn't make sense to merge then? It's maybe a bit unfortunate that this will be broken until the next staging cycle completes.